lambdas/data-migration2/tests/test-executions.js

```diff
@@ -0,0 +1,456 @@
+const { RecordAlreadyMigrated } = require('../dist/lambda/errors');
+
+const {
+  migrateExecutionRecord,
+  migrateExecutions,
+} = require('../dist/lambda/executions');
+
+let collectionsModel;
+let executionsModel;
+let asyncOperationsModel;
+let rulesModel;
+
+const executionOmitList = [
+  'createdAt', 'updatedAt', 'finalPayload', 'originalPayload', 'parentArn', 'type', 'execution', 'name', 'collectionId', 'asyncOperationId', 'cumulusVersion',
+];
+
+const testDbName = `data_migration_2_${cryptoRandomString({ length: 10 })}`;
+const testDbUser = 'postgres';
+
+const assertPgExecutionMatches = (t, dynamoExecution, pgExecution, overrides = {}) => {
+  t.deepEqual(
+    omit(pgExecution, ['cumulus_id']),
+    omit(
+      {
+        ...dynamoExecution,
+        async_operation_cumulus_id: null,
+        collection_cumulus_id: null,
+        parent_cumulus_id: null,
+        cumulus_version: dynamoExecution.cumulusVersion,
+        url: dynamoExecution.execution,
+        workflow_name: dynamoExecution.type,
+        original_payload: dynamoExecution.originalPayload,
+        final_payload: dynamoExecution.finalPayload,
+        created_at: new Date(dynamoExecution.createdAt),
+        updated_at: new Date(dynamoExecution.updatedAt),
+        timestamp: new Date(dynamoExecution.timestamp),
+        ...overrides,
+      },
+      executionOmitList
+    )
+  );
+};
+
+test.before(async (t) => {
+  process.env.stackName = cryptoRandomString({ length: 10 });
+  process.env.system_bucket = cryptoRandomString({ length: 10 });
+  process.env.ExecutionsTable = cryptoRandomString({ length: 10 });
+  process.env.AsyncOperationsTable = cryptoRandomString({ length: 10 });
+  process.env.CollectionsTable = cryptoRandomString({ length: 10 });
+  process.env.RulesTable = cryptoRandomString({ length: 10 });
+
+  await createBucket(process.env.system_bucket);
+
+  executionsModel = new Execution();
+  asyncOperationsModel = new AsyncOperation({
+    stackName: process.env.stackName,
+    systemBucket: process.env.system_bucket,
+  });
+  collectionsModel = new Collection();
+  rulesModel = new Rule();
+
+  await executionsModel.createTable();
+  await asyncOperationsModel.createTable();
+  await collectionsModel.createTable();
+  await rulesModel.createTable();
+
+  t.context.knexAdmin = await getKnexClient({
+    env: {
+      ...localStackConnectionEnv,
+      migrationDir,
+    },
+  });
+  await t.context.knexAdmin.raw(`create database "${testDbName}";`);
+  await t.context.knexAdmin.raw(`grant all privileges on database "${testDbName}" to "${testDbUser}"`);
+
+  t.context.knex = await getKnexClient({
+    env: {
+      ...localStackConnectionEnv,
+      PG_DATABASE: testDbName,
+      migrationDir,
+    },
+  });
+
+  await t.context.knex.migrate.latest();
+});
+
+test.afterEach.always(async (t) => {
+  await t.context.knex('executions').del();
+  await t.context.knex('collections').del();
+  await t.context.knex('async_operations').del();
```

Comment:
Same here, consider using this helper: https://github.com/nasa/cumulus/blob/feature-rds/packages/db/src/test-utils.ts#L38